src/platform-includes/crons/setup/python.mdx

```diff
@@ -1,12 +1,18 @@
 ## Job Monitoring
 
-Use the Python SDK to monitor and notify you if your job is missed (or doesn't start when expected), if it fails due to a problem in the runtime (such as an error), or if it fails by exceeding its maximum runtime.
+Use the Python SDK to monitor and notify you if your periodic task is missed (or doesn't start when expected), if it fails due to a problem in the runtime (such as an error), or if it fails by exceeding its maximum runtime.
+
+<Alert level="info" title="Note on Celery">
+
+If you are using **Celery Beat** to run you periodic tasks, have a look at our [Celery Beat Auto Discovery](/platforms/python/guides/celery/crons/)
+
+</Alert>
```

Comment:
```suggestion
<Alert level="info" title="Note on Celery">

If you're using **Celery Beat** to run you periodic tasks, have a look at our [Celery Beat Auto Discovery documentation](/platforms/python/guides/celery/crons/).

</Alert>
```

Reply:
edited your suggestion to fix a typo ("you" -> "your")